apps/web/src/server/app/validation/validation.controller.js

```diff
@@ -168,3 +169,16 @@ export function getIncompleteAppealOutcome(request, response) {
 		changeOutcomeURL: backURL
 	});
 }
+
+/**
+ * GET the incomplete appeal outcome next page journey.
+ *
+ * @param {object} request - Express request object
+ * @param {object} response - Express request object
+ * @returns {void}
+ */
+ export function getAppealOutcomeInvalidReason (request, response) {
```

Comment:
Can you please move the content of this func into [getInvalidAppealOutcome(request, response)](https://github.com/Planning-Inspectorate/back-office/blob/develop/apps/web/src/server/app/validation/validation.controller.js#L175).